src/content/docs/usage/workspaces.mdx

```diff
@@ -223,12 +328,16 @@ You can access services running in a Workspace on your local machine.
    daytona forward <PORT> <WORKSPACE> <PROJECT>
    ```
 
-   **Example: allow local access to port 4321 of the `example-dev-env` Workspace**
+   **Example:** Allow local access to port 4321 of the `example-dev-env` Workspace.
 
    ```shell
    daytona forward 4321 example-dev-env
    ```
 
+   Upon running this command, Daytona will establish a port forwarding connection from the specified port in the Workspace to your local machine or via a shareable public URL. The terminal will display a message indicating the local port where the forwarded connection is available:
```

Comment:
`via a shareable public URL` the link is public only if the user supplies the `--public` flag.